src/components/AddItem.jsx

```diff
@@ -22,79 +23,78 @@ const radioButtonOptions = [
   },
 ];
 
+function removePunctuation(string) {
+  const punctuationlessString = string
+    .toLowerCase()
+    .replace(/[^\w\s]|_/g, '')
```

Comment:
I love how this was pulled into a helper function, and I think putting it in a util file is a great idea! I know there's a way to adjust the regex so you don't have to use `.split` and `.join`. I tried using this instead `replace(/[^\w]|_/g` and it seemed to work! Based on this [Stack Overflow thread](https://stackoverflow.com/questions/16004314/remove-punctuation-and-spaces-from-string-using-regex/16004382), that seems to remove anything that's a non-word character.